src/scripts/visualizations/tooltip.js

```diff
@@ -38,6 +40,8 @@
             var plotTop = this.options.chart.plotTop;
             var plotHeight = this.options.chart.plotHeight;
             var distance = this.options.tooltip.distance;
+            var distanceX = this.options.tooltip.distanceX ? this.options.tooltip.distanceX : distance;
```

Comment:
I think this will fix it 
```
                var distanceX = this.options.tooltip.distanceX !== undefined ? this.options.tooltip.distanceX : distance;
                var distanceY = this.options.tooltip.distanceY !== undefined ? this.options.tooltip.distanceY : distance;
```